app/assets/stylesheets/partials/_registrations.scss

```diff
@@ -0,0 +1,9 @@
+.wcr-action-boxout {
```

Comment:
@irisfaraway PR #377 Also added actions to the dashboard, though she has a different back color (I can see from WEX that we've used [#f8f8f8](https://github.com/DEFRA/waste-exemptions-back-office/blob/master/app/assets/stylesheets/partials/_resource_details.scss#L2) there.

I'm just wondering if there is a way we can define the main class in one place to ensure some consistency, and then build on it as needed in the dashboard and view details page?

Reply:
Oh! I have missed that!! Will reuse that code! I think I did copy the colour and style from WEX entirely. Will also update the colour to match WEX :)